### PR TITLE
Make initialize database button easier to see on installer wizard #40

### PR DESCRIPTION
**Pull request by pjaudiomv**

### CODE REVIEW

**Comment by pjaudiomv** [comment]
screen shot 2018-11-29 at 12 06 52 pm
screen shot 2018-11-29 at 12 06 39 pm

**Comment by LittleGreenViper** [comment]
Might want to find out what kind of problem ol' Travis has with your checkin.

**Comment by pjaudiomv** [comment]
yea nothing to do with my pr, travis is slacking. having problems with docker.

**Comment by radius314** [comment]
What if the settings weren’t show until you mashed the “initialize” button?

This would make it so obvious, people would have trouble not seeing it.

**Comment by pjaudiomv** [comment]
Makes sense. You no mash, you no get settings.

**Comment by pjaudiomv** [comment]
screen shot 2018-11-29 at 7 43 01 pm

screen shot 2018-11-29 at 7 43 19 pm

**Comment by pjaudiomv** [comment]
hows this, or should i show the intro to settings file part as well before mashing button

File: main_server/local_server/install_wizard/installer.js [hunk 1]
```diff
@@ -464,6 +464,7 @@ function BMLTInstaller( in_prefs    ///< A JSON object with the initial prefs.
                     {
```
Comment (jbraswell) [inline main_server/local_server/install_wizard/installer.js]: It's good to get into the habit of using `===` in javascript. I'd change this to `=== true`.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Comparison_Operators

File: README.md [hunk 2]
```diff
@@ -64,6 +64,8 @@ CHANGELIST
 - Added user sort to full meeting list editors on service body administration.
 - Fix for default duration time not displaying correctly on meeting editor.
 - Added sort to meeting format checkboxes on formats page by format code.
+- Made initialize database button easier to see on installer wizard, and settings only gets displayed once button is mashed.
+- Added cache busting to installer wizard.
```
Comment (jbraswell) [inline README.md]: Will you be more specific about the files you add cache busting to? Like, say you added cache busting to the installer wizard's javascript and css files.

**Pull request merged**